Protocol.md

```diff
@@ -100,7 +100,7 @@ To enable the reuse of already-existing packfiles and indexes, a custom format f
 is supported. The `prefetch` endpoint will return one or more packfiles of **non-blob** objects.  
 
 If the optional `lastPackTimestamp` query parameter is supplied, only packs created by the server
-after the specific Unix epoch time (approximately, �10 minutes or so) will be returned. Generally, these packs 
+after the specific Unix epoch time (approximately, ±10 minutes or so) will be returned. Generally, these packs 

```

Comment:
I suspect this line was flagged as changed due to a character-encoding difference on the `±` symbol... see PR comment for more information.